contrib/functions/go/delete-annotations/README.md

```diff
@@ -0,0 +1,34 @@
+# delete-annotations
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Deletes the supplied annotation keys from the resource(s) in a package. It can delete multiple annotations provided as a comma separated string as part of the function config (usage instructions below).
+
+<!--mdtogo:Long-->
+
+## Usage
+
+The function will execute as follows:
+
+1. Searches for resources with valid metadata
```

Comment:
The steps here are not very useful, since what this function does is simple.

Reply:
Acked